docs/how-to/create-a-router/configure-enricher.md

```diff
@@ -10,7 +10,12 @@ The Turing enricher has the ability to perform arbitrary transformations on the
 Original request will be sent to configured routes without enrichment.
 
 ### Docker
-Turing will deploy specified Docker image as a pre-processor and will send original request to it for enrichment. To configure a Docker enricher, there are 3 sections to fill.  
+Turing will deploy specified Docker image as a pre-processor and will send original request to it for enrichment. 
+
+The request to the enricher is constructed from the request headers to the Router, and an identifier `turing_req_id` 
+that is uniquely assigned to each request received by the Router.
```

Comment:
`that is uniquely assigned to each request received by the Router` -> `that is uniquely assigned to each request "is" received by the "Enricher"`  ?

Also, isn't it `Turing-Req-ID` instead of `turing_req_id` ?

Reply:
Oh yeah, that's right; it should be `Turingn-Req-Id`